model_db.py

```diff
@@ -37,3 +39,26 @@ def init_from_disk(self, filepath: str = "models"):
             model = model_name.split("--")[-1]
             self[model] = SentenceTransformer(model_path)
             log.info(f"Loaded model {model}")
+
+    def embed_sentences(self, model: str, sentences: list[str]):
+        model_obj = self[model]
+        embeddings = model_obj.encode(sentences)
+
+        reformatted_embeddings = [
+            Embedding(
+                object="embedding",
+                index=i,
+                embedding=list(embedding),
+            )
+            for i, embedding in enumerate(embeddings)
+        ]
+
+        output = EmbeddingResponse(
+            object="list",
+            data=reformatted_embeddings,
+            embedding_id=str(uuid4()),
+            model=model,
+            model_info=self.get_model_info(model),
```

Comment:
I wonder if we need this `model` property if we're also giving back the`model_info` that includes the model name as well?

Reply:
We can leave it in by all means if you think it would be useful by consuming functions, just spitballing

Reply:
@lmwilki will explain! (i think this is a standard way of representing the response)